typescript/ecs/ecs-service-with-logging/package-lock.json

```diff
@@ -5,14 +5,12 @@
   "requires": true,
```

Comment:
can you actually delete this file from the commit? It gets regenerated when you run npm install.